examples/data-sources/powermax_storagegroup/data-source.tf

```diff
@@ -16,7 +16,7 @@ limitations under the License.
 */
 data "powermax_storagegroup" "test" {
   filter {
-    names = ["esa_sg572"]
+    names = ["example_sg"]
```

Comment:
@doriac11 Please check if UTs and FTs are working correctly after these changes.

Reply:
They are @nikitajoshi1 , this is only in the example file, not the test itself.